go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/shopspring/decimal
 
-go 1.13
+go 1.19
```

Comment:
Go version in `go.mod` specifies the minimum Go version required to compile a given module, thus I don't think we should change it to 1.19. Decimal can be run with go1.7+